idd/Energy+.idd.in

```diff
@@ -33053,6 +33053,12 @@ ZoneHVAC:IdealLoadsAirSystem,
        \note This field is optional, but is required if this
        \note this object is used with other forced air equipment.
        \type node
+  A5 , \field System Inlet Air Node Name
+       \note When the Ideal Loads Air System is connected to an AirloopHVAC:ZoneReturnPlenum the
+       \note plenum exhaust node will connect to this System Inlet Node while the Zone Supply
+       \note Air Node specified above will identify the zone inlet node where this Ideal Loads
+       \note Air System is connected. The Zone Exhaust Air Node will be the same as the
+       \note AirloopHVAC:ReturnAirPlenum inlet node.
```

Comment:
This needs to be very clear that it's only required *if* the ideal loads system is connected to a return plenum.
Other edits:
the plenum exhaust node --> a plenum induced air node
The Zone Exhaust Air Node will be the same as the AirloopHVAC:ReturnAirPlenum inlet node --> same as *a* . . .